packages/@aws-cdk/aws-rds/lib/instance-engine.ts

```diff
@@ -153,49 +513,246 @@ class PostgresInstanceEngine extends InstanceEngineBase {
       engineType: 'postgres',
       singleUserRotationApplication: secretsmanager.SecretRotationApplication.POSTGRES_ROTATION_SINGLE_USER,
       multiUserRotationApplication: secretsmanager.SecretRotationApplication.POSTGRES_ROTATION_MULTI_USER,
-      parameterGroupFamilies: [
-        { engineMajorVersion: '9.3', parameterGroupFamily: 'postgres9.3' },
-        { engineMajorVersion: '9.4', parameterGroupFamily: 'postgres9.4' },
-        { engineMajorVersion: '9.5', parameterGroupFamily: 'postgres9.5' },
-        { engineMajorVersion: '9.6', parameterGroupFamily: 'postgres9.6' },
-        { engineMajorVersion: '10',  parameterGroupFamily: 'postgres10'  },
-        { engineMajorVersion: '11',  parameterGroupFamily: 'postgres11'  },
-      ],
-      version: props.version,
+      version: props.version
+        ? {
+          fullVersion: props.version.postgresFullVersion,
+          majorVersion: props.version.postgresMajorVersion,
+        }
+        : undefined,
     });
   }
 }
 
-interface OracleInstanceEngineProps extends InstanceEngineProps {
+/**
+ * The versions for the legacy Oracle instance engines
+ * (those returned by {@link DatabaseInstanceEngine.oracleSe}
+ * and {@link DatabaseInstanceEngine.oracleSe1}).
+ * Note: RDS will stop allowing creating new databases with this version in August 2020.
```

Comment:
Is it worth pushing this only to deprecate it in one month?

Reply:
I think so. People still might have CDK code with existing databases using it.

Reply:
They can use the `of()` then?